menubar.js

```diff
@@ -0,0 +1,65 @@
+const fs = require("node:fs");
+
+module.exports = [
+    {
+        label: 'App',
+        submenu: [
+            {
+                label: 'Quit',
+                accelerator: 'ctrl+q',
```

Comment:
Can we use `CommandOrControl+q` to support common Mac, Windows and Linux shortcuts? This change would occur in all menu items currently using `ctrl+`